### PR TITLE
Add comment on customer

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR adds a possibility to add comments on customers

**How to install**:
1. install on clinical-api-stage of the clinical-db machine: `bash servers/resources/clinical-db.scilifelab.se/update-clinical-api-stage.sh add-comment-on-customer`
1. run script cg/scripts/add-comment-to-customer.sql on the cg-stage db

**How to test**:
1. login to clinical-api-stage
1. open customer view
1. click on empty on a customer in the comment column
1. add a comment

**Expected outcome**:
- [x] comment should be visible
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by @ingkebil 
- [x] tests executed by @patrikgrenfeldt 
- [x] "Merge and deploy" approved by @ingkebil 
Thanks for filling in who performed the code review and the test!

This is minor **version bump** because we add func in backwards compatible manner

### CODE REVIEW

**Comment by ingkebil** [comment]
Code looks good!

File: scripts/add-comment-to-customer.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE customer ADD COLUMN comment text;
```
Comment (ingkebil) [inline scripts/add-comment-to-customer.sql]: Do we need to set the default value to something or make it nullable?

Comment (patrikgrenfeldt) [reply scripts/add-comment-to-customer.sql]: seems not: 
image

**Comment by patrikgrenfeldt** [comment]
Test result:
image
image

**Comment by patrikgrenfeldt** [comment]
@ingkebil may I deploy?

**Comment by ingkebil** [comment]
Go ahead!

**Pull request merged**